docs/operator-manual/metrics.md

```diff
@@ -3,7 +3,7 @@
 Argo CD exposes different sets of Prometheus metrics per server.
 
 ## Application Controller Metrics
-Metrics about applications. Scraped at the `argocd-metrics:8082/metrics` endpoint.
+Metrics about applications. Scraped at the `argocd-application-controller-metrics:8082/metrics` endpoint.
```

Comment:
@mkilchhofer Do you think this could be a little confusing? `argcd-application-controller-metrics` sounds like this could include metrics about the app controller's performance rather than application related metrics